w3lib/encoding.py

```diff
@@ -143,21 +160,23 @@ def resolve_encoding(encoding_alias):
 ]
 _FIRST_CHARS = set(c[0] for (c, _) in _BOM_TABLE)
 
+
 def read_bom(data):
+    # type: (bytes) -> Tuple[str, bytes]
     r"""Read the byte order mark in the text, if present, and
     return the encoding represented by the BOM and the BOM.
 
     If no BOM can be detected, ``(None, None)`` is returned.
```

Comment:
Ah, didn't know that, thanks! mypy behavior makes sense for tuple, although not that I would expect it in general :)